github/packages/github/github_close_issue.js

```diff
@@ -0,0 +1,68 @@
+// jshint esversion: 9
+
+/**
+ * @description null
+ * @param {ParamsType} params list of command parameters
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+const axios = require('axios');
+
+async function closeIssue(url, secrets) {
+
+    return (axios({
+        method: "patch",
+        url: url,
+        data: {state: 'closed'},
+        headers: {
+            Authorization: `Bearer ${secrets.github_token}`,
+            "Content-Type": "application/json"
+        }})
+        .then(res => { return res.data; })
+        .catch(err => { return err; }));
+}
+
+async function _command(params, commandText, secrets = {}) {
+  
+  if (!secrets.github_token) {
+    return {
+      response_type: 'in_channel',
+      text: 'Missing Github Personal Access Token!'

```

Comment:
Something like this would be more helpful to the user: https://github.com/nimbella/command-sets/blob/f5668d0d698a6fde793b085c9aaaee74924807fe/do/packages/do/droplet_list.js#L98